src/EditorFeatures/Core/Implementation/IntelliSense/Completion/CompletionFilterReason.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Completion
+{
+    internal enum CompletionFilterReason
+    {
+        Insertion,
+        Deletion,
+        NonInsertionOrDeletion,
+#if false
+        // If necessary, we could add additional filter reasons.  For example, for the below items.
```

Comment:
This comment feels much like "I'm commenting out code I thought I needed". Perhaps rewrite to more clear prose?